package.json

```diff
@@ -1,197 +1,176 @@
 {
```

Comment:
Please don't modify `package.json`

Reply:
Sorry :( It was my linter. I'll revert the changes and push only the dependency I need!